scripts/utils/generate_ts.sh

```diff
@@ -58,7 +58,7 @@ mkdir -p addon_ts || die
 cp addons/generated/addons/*.ts addon_ts
 print G "done."
 
-for branch in $(git branch -r | grep origin/releases); do
+for branch in $(git branch -r | grep origin/releases/2.15.*); do
```

Comment:
Do we need to add this in? Without it, we just get a longer list of release branches - including 2.15.*.

Reply:
It fails once we get to releases/2.16 when `strings.yaml` isn't found at `translations/strings.yaml` and (afaik) we don't need to import strings from that branch.

Reply:
Just to confirm, this is a one-off problem, because of #6401? It's not a problem going forward to build from branches?

Reply:
That's correct. We have patched `generate_ts.sh` in 2.16 to handle the differences between branches.